packages/now-cli/src/util/print-indications.ts

```diff
@@ -0,0 +1,32 @@
+import chalk from 'chalk';
+import { Response } from 'node-fetch';
+import { emoji, EmojiLabel, prependEmoji } from './emoji';
+import createOutput from './output';
+import linkStyle from './output/link';
+
+export default function printIndications(res: Response) {
+  const _output = createOutput();
+  const indications = new Set(['warning', 'notice', 'tip']);
+  const regex = /^x-(?:vercel|now)-(warning|notice|tip)-(.*)$/;
```

Comment:
Do we want to keep supporting `now` here? I think we could only check for `x-vercel` here.

Reply:
I left it in case we still have any API endpoints which return `x-now`.